<ui version="4.0" >
 <class>qikeaConflict</class>
 <widget class="QDialog" name="qikeaConflict" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Naming Conflict Detected</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>:/png/ikea.png</normaloff>:/png/ikea.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="textConflictMessage" >
     <property name="text" >
      <string>A file with the same name already exists. Please select overwrite or enter a different name to continue.</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineConflictName" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOverwrite" >
       <property name="text" >
        <string>&amp;Overwrite</string>
       </property>
       <property name="shortcut" >
        <string>Alt+O</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
       <property name="default" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonContinue" >
       <property name="text" >
        <string>&amp;Continue</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ikea.qrc" />
 </resources>
 <connections/>
</ui>
